common/src/IO/AssimpParser.h

```diff
@@ -24,7 +24,7 @@
 #include "Assets/EntityModel_Forward.h"
 #include "IO/EntityModelParser.h"
 
-#include <assimp/matrix4x4.h>
+#include <assimp/types.h>
```

Comment:
Why do you need this change in the header file?

Reply:
`aiString` is defined in this header only, which is used in the new `AssimpBoneInformation` struct.

Reply:
Yeah, but you don't use it in this header file, right?